tensorflow/examples/speech_commands/recognize_commands.py

```diff
@@ -0,0 +1,87 @@
+from collections import deque
+import numpy as np
+
+class RecognizeResult:
```

Comment:
Here, and other places: Inherit classes from `object`, i.e. `class RecognizeResult(object):`